### ISSUE TITLE
Publication of OSM Paper 1

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSM_To_Do_List#434]
The first paper to arise from the Open Source Malaria Consortium is [published today in ACS Central Science](http://pubs.acs.org/doi/full/10.1021/acscentsci.6b00086), describing Series 1. Congratulations again to all those OSMers who were involved with this one.

This Issue can act as a place to gather together any materials related to this publication that might be of interest to contributors here. Please add any others below. I’ll close the Issue in a couple of weeks and port the resources to the wiki.

1. [Paper](http://pubs.acs.org/doi/full/10.1021/acscentsci.6b00086 ) (includes links out to all the supporting information etc)
2. [Lay article in The Conversation](https://theconversation.com/making-drug-development-less-secretive-could-lead-to-quicker-cheaper-therapies-64744)
3. [Blog Post](https://intermolecular.wordpress.com/2016/09/14/open-source-malarias-first-paper/) by Mat about the project
4. [Article in El Mundo](http://www.elmundo.es/salud/2016/09/14/57d81820ca47410b2c8b466f.html).

**Comment by cdsouthan** [comment]
Brill,  when the paper goes live in PubMed I will briefly point to Mat's and my blogposts as a PubMed Commons entry.  I suggest Mat adds any new pointers of significance as updates to his own blog so we knit the commentaries together.

**Comment by cdsouthan** [comment]
This is obvious but could we encourage anyone to attempt to identify the Series1 pathway and/or molecular target (including eliminating Pf4ATPase)?

cf https://cdsouthan.blogspot.se/2012/12/antimalarial-target-deconvolution.html

**Comment by mattodd** [comment]
Absolutely. This is probably the number 1 thing on the Series 1 to do list. In the paper we eliminate a couple of possibilities (DHODH, PfATP4) but Iain Wallace made predictions on others that have not yet been screened: "carboxy-terminal domain RNA polymerase II polypeptide A small phosphatase 1 (Q8I3U9), SUMO-activating enzyme subunit 2 (Q8I553) and 1 (Q8IHS2), and cyclin-dependent kinase 1 (P61075)."
Are there other targets you were thinking of?
The alternative is generating resistant mutants. There are labs that do this, but the work costs.

**Comment by cdsouthan** [comment]
Good, 
JFTR I think I covered most of  the options in  https://cdsouthan.blogspot.se/2012/12/antimalarial-target-deconvolution-part.html

The two main things that have changed in the last 4 years are the cost of RNA seq (transcript profling for mmoa fingeprintin) and of course CRISPR for dialing mutants.  Testing the in silco predictions would depend on finding in vitro assys for those enzymes
